docs/Dockerfile

```
@@ -7,11 +7,27 @@ COPY content content
 COPY tools tools
 COPY Makefile ./
 
-RUN npm install -g npm@8.1.4 && npm install -g yarn@1.22.17
+RUN npm install -g npm@8.9.0 && npm install -g yarn@1.22.18
 RUN make
 
 FROM registry.access.redhat.com/ubi8/nginx-120:latest
 
+RUN echo "$(id -u):$(id -g)" > /tmp/container-uid
+
+# Update RPMs to hopefully aid with vulns that aren't yet patched in the base image. Also remove package management
+# utils because our own policies don't recommend having them in the image. For that we need to be root.
+USER 0:0
+RUN dnf upgrade -y && \
+    dnf clean all && \
+    rpm --verbose -e --nodeps $(rpm -qa curl '*rpm*' '*dnf*' '*libsolv*' '*hawkey*' 'yum*') && \
+    rm -rf /var/cache/dnf /var/cache/yum
+
+# Switch back to the normal user of nginx container and check that we really switch to the original one.
+USER 1001:0
+RUN echo "If the following command fails, update USER statement to match UID:GID of the user of nginx base container." && \
+    [ "$(id -u):$(id -g)" = "$(cat /tmp/container-uid)" ] && \
```

maybe a single `=` works in sh but it looks wrong to me. 
```suggestion
    [ "$(id -u):$(id -g)" == "$(cat /tmp/container-uid)" ] && \
```

---

The thing is: I used `test` (`[`) external command but not bash built-in `[[`. I believe, `test`-s equality check is only `=`. From `man test`:
```
       STRING1 = STRING2
              the strings are equal
```

I checked that `[[` works in that command too, so using now that.